kobo/apps/audit_log/tests/test_project_history_logs.py

```diff
@@ -862,3 +880,43 @@ def test_export_v1_creates_log(self):
         log = log_query.first()
         self._check_common_metadata(log.metadata, PROJECT_HISTORY_LOG_PROJECT_SUBTYPE)
         self.assertEqual(log.object_id, self.asset.id)
+
+    @data(
+        ('archive', AuditAction.ARCHIVE, False),
+        ('unarchive', AuditAction.UNARCHIVE, False),
+        ('undelete', AuditAction.UNDELETE, True),
```

Comment:
I would just use 'None' here for the action so we don't need to make an AuditAction.UNDELETE that we don't actually use

Reply:
Ok, good idea